### PR TITLE
Adding aggregate_Ushauri_sms_reminders

### PR DESCRIPTION
**Pull request by Marymary-dev**
https://thepalladiumgroup.atlassian.net/browse/KHP3-4806

### CODE REVIEW

File: Scripts/REPORTING/load_aggregate_Ushauri_sms_reminders.sql [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+         ConsentForSMS = 'YES' 
+      GROUP BY
+         EOMONTH (Try_CONVERT(date, AppointmentDate)),
+         MFLCode 
+   )
+,
+   ReceivedSMS AS 
+   (
+      SELECT
+         COALESCE( Try_CONVERT(date, FourWeekSMSSendDate), Try_CONVERT(date, ThreeWeekSMSSendDate), Try_CONVERT(date, TwoWeekSMSSendDate), Try_CONVERT(date, OneWeekSMSSendDate), Try_CONVERT (date, OneDaySMSSendDate) ) AS AsofDate,
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_Ushauri_sms_reminders.sql:42]: @Marymary-dev for readability maybe we can have each column in a newline.

File: Scripts/REPORTING/load_aggregate_Ushauri_sms_reminders.sql [hunk 2]
```diff
@@ -0,0 +1,134 @@
+IF OBJECT_ID(N'[Reporting].[Dbo].[aggregateUshauriSMSReminders]', N'U') IS NOT NULL DROP TABLE [Reporting].[Dbo].[aggregateUshauriSMSReminders];
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_Ushauri_sms_reminders.sql:1]: @Marymary-dev nitpik: Maybe we can rename to be in CamelCase e.g. `AggregateUshauriSMSReminders`

consider newline begining `DROP ...` :)

**Review by nobert-mumo — CHANGES_REQUESTED** [review]

File: Scripts/REPORTING/load_aggregate_Ushauri_sms_reminders.sql [hunk 3]
```diff
@@ -0,0 +1,97 @@
+IF Object_id(N'[Reporting].[Dbo].[AggregateUshauriSMSReminders]', N'U') IS NOT NULL
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_Ushauri_sms_reminders.sql:1]: @Marymary-dev do we want to only show disaggregation by AsOfDate or also the other standard dimensions e.g Facility, AgeGroup, Partner, Agency etc ?

**Comment by Marymary-dev** [comment]
@nobert-mumo , @cbrianbet  kind reminder to review this

**Comment by nobert-mumo** [comment]
> @nobert-mumo , @cbrianbet kind reminder to review this

Sorry @Marymary-dev missed out on this; this is probably affected by the PatientPK in Ushauri discussions.?

**Comment by Marymary-dev** [comment]
Yes it would be affected

File: Scripts/REPORTING/load_aggregate_Ushauri_sms_reminders.sql [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+           Coalesce (Honouredappointments.Numberhonouredappointment, 0)
+           As
+           NumberHonouredAppointment,
+           Coalesce(Cast(Honouredappointments.Numberhonouredappointment As Float
+                    ) /
+                    Nullif(
+                             Appointmentcounts.Totalappointments, 0) * 100, 0)
+           As
+           PercentHonoured
+    Into   reporting.dbo.Aggregateushaurismsreminders
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_Ushauri_sms_reminders.sql:109]: nitpik: we can have as CamelCase e.g. `AggregateUshauriSmsReminders`

**Review by nobert-mumo — APPROVED** [review]

**Pull request merged**